report.md

```diff
@@ -1,7 +1,60 @@
 # Subsystem Design Report
 
 ## Subsystem 1 (MFA)
-
+
+MFA Tag Interfaces:
+1. RFDuino BLE API  
+The RFduino device must include the #include <RFduinoBLE.h> command in order to make use of the RFduino BLE. 
+1. Arduino Cryptography Libraries
+
+Web App Interface:
+1. Web Bluetooth API  
+Access to Web Bluetooth API will be subject to permissions and it will only work in secure contexts. It only works with sites on HTTPS.  The connection to the device can only be called by a  user action like a click for example. 
+
+Communication protocol between the two devices:  
+Digital Signature + Diffie Hellman Key Exchange + Symmetric Encryption
```

Comment:
I think this one should remove Diffie Hellman Key Exchange? Unless it is confirmed to be this with  no underlying extra steps?

Reply:
ok! cos i found the curve25519 - that one use diffie hellman key exchange but okay i will just write "key exchange"